README.md

```diff
@@ -21,9 +21,18 @@ Variables without a default value and not marked optional, *MUST* be defined for
 
 | Environment Variable               | Default                   | Description 
 | -----------------------------------|---------------------------|-------------
-| GP2GP_SERVER_PORT                  | 8080                      | The port on which the SCR API will run
+| GP2GP_SERVER_PORT                  | 8080                      | The port on which the SCR API will run.
 | GP2GP_LOGGING_LEVEL                | INFO                      | Application logging level. One of: DEBUG, INFO, WARN, ERROR. The level DEBUG **MUST NOT** be used when handling live patient data.
-| GP2GP_LOGGING_FORMAT               | (*)                       | Defines how to format log events on stdout
+| GP2GP_LOGGING_FORMAT               | (*)                       | Defines how to format log events on stdout.
+| GP2GP_MONGO_URI                    | mongodb://localhost:27017 | Whole Mongo database connection string. Has a priority over other Mongo variables.
+| GP2GP_MONGO_DATABASE_NAME          | gp2gp                     | Mongo database name.
+| GP2GP_MONGO_HOST                   | (*)                       | Mongo database host. Can be left blank if full connection string is provided.
+| GP2GP_MONGO_USERNAME               | (*)                       | Mongo database username. Can be left blank if full connection string is provided.
+| GP2GP_MONGO_PASSWORD               | (*)                       | Mongo database password. Can be left blank if full connection string is provided.
+| GP2GP_MONGO_OPTIONS                | (*)                       | Mongodb URL encoded parameters for the connection string without a leading "?". Can be left blank if full connection string is provided.
+| GP2GP_MONGO_AUTO_INDEX_CREATION    | true                      | (Optional) Should auto index for Mongo database be created.
+| GP2GP_MONGO_TTL                    | P30D                      | (Optional) Time-to-live value for inbound and outbound state collection documents as an [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations).
+| GP2GP_COSMOS_DB_ENABLED            | false                     | (Optional) If true the adaptor will enable features and workarounds to support Azure Cosmos DB.
 
```

Comment:
Please add necessary variables added in the environment variable list to the section below 'How to run service'